provider/inspec.rb

```diff
@@ -60,6 +61,32 @@ def generate_resource(data)
       generate_properties(data.clone, data.object.all_user_properties)
     end
 
+    # Generate the IAM policy for this object. This is used to query and test
+    # IAM policies separately from the resource itself
+    def generate_iam_policy(data)
```

Comment:
I think this method should accept `output_folder` or some other overridable method. Right now it's a pretty inspec specific in where it will be generating the output code.

Reply:
This is inside the inspec.rb file, so this is intended?

Reply:
derp. nm them